### DISCUSSION TITLE
Unable to login after upgrade

### DISCUSSION THREAD

**Discussion by Welfordian** [filamentphp/filament#7495] [General]
Hi all,

After upgrading to Filament v3, I am unable to login to my panel.
It is hosted on a sub-directory `/sub/admin`.

I have set the following configuration:-

`->path('/sub/admin')`

```
Livewire::setScriptRoute(function ($handle) {
    return Route::get('/sub/livewire/livewire.js', $handle);
});

Livewire::setUpdateRoute(function ($handle) {
    return Route::post('/sub/livewire/update', $handle);
});
```

I _was_ logged in after I upgraded and the page just kept reloading over & over again.
After logging out, I can no longer login.

Any ideas?

Thanks!

**Comment by Welfordian** [comment]
After blocking the redirect loop, I saw this in the console:-

```
Uncaught (in promise) SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at livewire.js?id=99a05389:269:46
    at async queueNewRequestAttemptsWhile (livewire.js?id=99a05389:313:5)
    at async sendRequestToServer (livewire.js?id=99a05389:207:5)
```

**Comment by Welfordian** [comment]
This has temporarily solved the issue, but none of the livewire functionality works.

```
    // Illuminate\Foundation\Exceptions\Handler;
    protected function unauthenticated($request, AuthenticationException $exception)
    {
        //        return $this->shouldReturnJson($request, $exception)
        //                    ? response()->json(['message' => $exception->getMessage()], 401)
        //                    : redirect()->guest($exception->redirectTo() ?? route('login'));
    }
```

**Comment by Welfordian** [comment]
I was missing `->middleware('web')` on the `setUpdateRoute`. What a facepalm.

**Comment by danharrin** [comment]
Please don't host a Laravel project like this, it usually just leads to issues across the whole framework. Subdomains are your friend.

**Reply by Welfordian** [reply]
Oh, I'm aware. Unfortunately, I'm not the one making these decisions.

**Reply by danharrin** [reply]
Tell whoever is in charge that Filament doesnt officially support being in a subdirectory like that, and will not fix bugs as a result of it
